seguno.com.domain-authentication.json

```diff
@@ -0,0 +1,25 @@
+{
+  "providerId": "Seguno.com",
+  "providerName": "Seguno",
+  "serviceId": "domain-authentication",
+  "serviceName": "Seguno Domain Authentication",
+  "version": 1,
+  "syncPubKeyDomain": "seguno.com",
+  "logoUrl": "https://cdn.seguno.com/images/seguno-logo.png",
+  "description": "Places SPF and DKIM records to authenticate email sent by Seguno Software on behalf of the user",
```

Comment:
for SPF I would have expected SPFM record. So either the description is wrong or the record is missing

Reply:
Thanks @pawel-kow, this was an incorrect description. It's not an SPF record but one used for [Return-Path/bounce domain](https://support.sparkpost.com/docs/tech-resources/custom-bounce-domain). We use Sparkpost as a sending platform and the SPF is handled automatically.